db/seeds.rb

```diff
@@ -29,11 +31,19 @@ def generate_diets
 		end
 	end
 
+	def generate_cuisines
+		["American", "Italian", "Asian", "Mexican", "Southern & Soul Food", "French", "Southwestern", "Barbecue", "Indian", "Chinese", "Cajun & Creole", "English", "Mediterranean", "Greek", "Spanish", "German", "Thai", 
```

Comment:
Line is too long. [213/80]<br>Use 2 (not 1) spaces for indentation.<br>Tab detected.<br>Trailing whitespace detected.